sonar-project.properties

```diff
@@ -1,18 +1,14 @@
-sonar.projectKey=metamask-extension
-sonar.organization=consensys
+sonar.projectKey=metamask-extension-private
```

Comment:
Question: In order to have no confusion and duplication, are we going to remove project with name of `metamask-extension` ? and if so, can you grand access to devs & qas in platform team so we can add more metrics 🙏🏻 thank you

Reply:
Hi Danica, no. I will update the description sorry... anyway I will reach out to you via Slack.